grub-efi/docker-create-grub-efi-binaries

```diff
@@ -29,12 +29,10 @@ for arch in x86-64 arm aarch64; do
     echo -n "Extracting binaries for $arch..."
     docker run --rm --entrypoint "/bin/sh" \
         -v $(pwd)/output/$arch:/export grub-efi:$arch \
-        -c "cp -r /install/* /export"
+        -c "chown -R --reference /export /install/* && mv /install/* /export"
     echo " done"
 done
 
-sudo chown -R $(id -u):$(id -g) output/*
-
```

Comment:
Nice trick! Thanks.

Please remove it also from [the other script](https://github.com/mendersoftware/mender-convert-integration-scripts/blob/master/docker-create-integration-binaries#L29) as well as for the install dependencies in GitLab Pipeline ([here](https://github.com/mendersoftware/mender-convert-integration-scripts/blob/master/.gitlab-ci.yml#L25) and [here](https://github.com/mendersoftware/mender-convert-integration-scripts/blob/master/.gitlab-ci.yml#L39))

Reply:
Done!